scripts/mkcloud

```diff
@@ -353,10 +353,15 @@ function onhost_create_cloud_lvm()
         _lvcreate $cloud.node$i $hdd_size $cloudvg $next_pv_device
     done
     if [ $controller_raid_volumes -gt 1 ] ; then
+        # total wipeout of the disks used for RAID, to prevent bsc#966685
+        volume="/dev/$cloudvg/$cloud.node1"
+        dd if=/dev/zero of=$volume bs=1G count=${controller_hdd_size}
```

Comment:
In my experience adding the flag 'oflag=sync' or && sync is better. Sometimes the bits/bytes are still in the `buffer` (may be wrong with the jargon here) and take some time to flush out (write to disk)! Not sure why this happens on some systems but it does happen. Same for line 364.